use failure::Error;
use std::env;

pub fn main() -> Result<(), Error> {
    for input_dir in env::args().skip(1) {
        
    }
    Ok(())
}
